doc/tutorials/launch_uos.sh

```diff
@@ -313,7 +312,7 @@ ACRN project
 intr_storm_monitor="--intr_monitor 10000,10,1,100"
 
  acrn-dm -A -m $mem_size -c $2 -s 0:0,hostbridge -s 1:0,lpc -l com1,stdio $npk_virt\
-   -s 9,virtio-net,$tap \
+   -s 9,virtio-net,tap=$6,mac_seed=$mac_seed  \
```

Comment:
Same comment here, shouldn't this be `tap_$6`?

Reply:
The $tap is the full device name which spliced by the pre str "tap_" and the parameter "$6".
So the type is tap and the name is $6.

Reply:
I'm a bit confused by all this, so our code makes an assumption that while the user passes a name such as `LaaG` (or any other name), the name of the actually TAP device must be `tap_LaaG`??

Reply:
The actually  device name is type_name which follow the old devices name format.

That means the old command  "-s 9,virtio-net,macvtap_LaaG" should change to "-s 9,virtio-net,macvtap=LaaG".

The actually devices is macvtap_LaaG.

Reply:
This is not what we should do, the previous mechanism forced the user to include `tap` in the device name so it could be properly identified. The idea is to move away from that restriction (on the name) and be able to use any name. This is why we now have to specify the type of device as well. As an example: `-s 9,virtio-net,tap=LaaG` means that User VM will connect to a TAP device called `LaaG`.

Reply:
I have resolved other open, let's focus on the "tap0".

The current patch have not change the create device logic.

So if user could create a device name "tab_xxx" as the current design, it's effective also.

Reply:
Same problem here